Packs/IllusiveNetworks/Playbooks/playbook-Illusive-Retrieve-Incident.yml

```diff
@@ -0,0 +1,260 @@
+    - label: yes
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              complex:
+                root: Illusive
+                accessor: Incident.hasForensics
+            iscontext: true
+          right:
+            value:
+              simple: 'true'
+    view: "{\n  \"position\": {\n    \"x\": 450,\n    \"y\": 370\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '3':
+    id: '3'
+    taskid: c24b2463-4642-4d69-8209-66f425ede653
+    type: regular
+    task:
+      id: c24b2463-4642-4d69-8209-66f425ede653
+      version: -1
+      name: illusive-get-forensics-timeline
+      description: Retrieve forensics timeline for a specific incident
+      script: IllusiveNetworks|||illusive-get-forensics-timeline
+      type: regular
+      iscommand: true
+      brand: IllusiveNetworks
+    nexttasks:
+      '#none#':
+      - '6'
+    scriptarguments:
+      end_date:
+        complex:
+          root: inputs.end_date
+      incident_id:
+        complex:
+          root: Illusive
+          accessor: Incident.incidentId
+      start_date:
+        complex:
+          root: inputs.start_date
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 450,\n    \"y\": 570\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '5':
+    id: '5'
+    taskid: 07ef76cf-d732-4051-8b3b-87d60059e1b7
+    type: regular
+    task:
+      id: 07ef76cf-d732-4051-8b3b-87d60059e1b7
+      version: -1
+      name: Print No Forensics Timeline for Incdient
+      description: Prints text to war room (Markdown supported)
+      scriptName: Print
+      type: regular
+      iscommand: false
+      brand: ''
+    scriptarguments:
+      value:
+        simple: '"The Incident Has No Forensics'
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 1040,\n    \"y\": 560\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '6':
+    id: '6'
+    taskid: c8f99fa8-9ba3-4347-8aee-bbd05ef093d1
+    type: regular
+    task:
+      id: c8f99fa8-9ba3-4347-8aee-bbd05ef093d1
+      version: -1
+      name: Print Incident Forensics Timeline
+      description: Prints text to war room (Markdown supported)
+      scriptName: Print
+      type: regular
+      iscommand: false
+      brand: ''
+    scriptarguments:
+      value:
+        simple: ${Illusive.Forensics.Evidence}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 450,\n    \"y\": 740\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '7':
+    id: '7'
+    taskid: 8846a6d7-5c88-4001-8f9c-c076d8b6a31c
+    type: condition
+    task:
+      id: 8846a6d7-5c88-4001-8f9c-c076d8b6a31c
+      version: -1
+      name: Check Initial Values
```

Comment:
you can replace this steps by checking the input of a playbook to be mandatory
![image](https://user-images.githubusercontent.com/7270217/81586105-bf8cd000-93bd-11ea-873d-4e232ae27ef0.png)
![image](https://user-images.githubusercontent.com/7270217/81586163-cd425580-93bd-11ea-9f61-70cd58ead0b4.png)